lib/instrumentation/modules/cassandra-driver.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+/**
+ * TODO: Support batch queries
+ */
+
+const semver = require('semver')
+const sqlSummary = require('sql-summary')
+
+const shimmer = require('../shimmer')
+
+module.exports = function (cassandra, agent, version, enabled) {
+  if (!enabled) return cassandra
+  if (!semver.satisfies(version, '^3.0.0')) {
+    agent.logger.debug('cassandra version %s not supported - aborting...', version)
```

Comment:
I would probably write the full module name here (s/cassandra/cassandra-driver/) just to be consistent 😃

Reply:
done.